### PR TITLE
JR2-238 Create data model for candidate search request

### PR DESCRIPTION
**Pull request by varusakov**

### CODE REVIEW

File: src/main/webapp/app/candidate-search/services/candidate-search-request.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+export interface CandidateSearchRequest {
+    occupation: string,
+    skills: Array<string>,
+    experience: string,
+    workplace: string,
+    residence: string,
+    availability: string,
+    workload: number,
+    workSystem: string,
```
Comment (michael-wirth) [inline src/main/webapp/app/candidate-search/services/candidate-search-request.ts]: this has been renamed to `workForm` see https://github.com/alv-ch/candidate-service/blob/develop/src/main/java/ch/admin/seco/service/candidate/domain/CandidateProfile.java

File: src/main/webapp/app/candidate-search/services/candidate-search-request.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+export interface CandidateSearchRequest {
+    occupation: string,
+    skills: Array<string>,
+    experience: string,
+    workplace: string,
+    residence: string,
+    availability: string,
```
Comment (michael-wirth) [inline src/main/webapp/app/candidate-search/services/candidate-search-request.ts]: create enum for `availability`, `workForm`, `educationLevel`, `graduation`, `drivingLicenseCategory` and `CEFR_Level` = language-level

`graduation` is not yet defined on the server side. Just create an empty `enum`

Comment (varusakov) [reply src/main/webapp/app/candidate-search/services/candidate-search-request.ts]: Done

File: src/main/webapp/app/candidate-search/services/candidate-search-request.ts (hunk 1 above)
Comment (varusakov) [reply src/main/webapp/app/candidate-search/services/candidate-search-request.ts]: Renamed to `workForm`

**Pull request merged**